lib/grape/validations/types/primitive_coercer.rb

```diff
@@ -55,6 +55,12 @@ def reject?(val)
             (val.is_a?(String) && type == Hash) ||
             (val.is_a?(Hash) && type == String)
         end
+
+        # Dry-Types treats an empty string as invalid. However, Grape community wants to have
```

Comment:
I think we should explain why and link to the issue. How about: "However, Grape considers an empty string as absence of value and coerces it into `nil`."?